frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -8,7 +8,9 @@ local logger = require("logger")
 local T = require("ffi/util").template
 local _ = require("gettext")
 
-local ReaderCoptListener = EventListener:extend{}
+local ReaderCoptListener = EventListener:extend{
+    additional_header_content = {} -- place, where additional header content can be inserted.
```

Comment:
I'm not sure this is a good idea. When we had such stuff, it would leak between ReaderUI and caused hard to debug issues.
Best to create a new table in :init(), instead of having a permanent one as it would happen here. (This is fine for number and strings.)